blueman/bluez/Adapter.py

```diff
@@ -10,7 +10,7 @@ class Adapter(Base):
     _interface_name = 'org.bluez.Adapter1'
 
     def __init__(self, obj_path: str):
-        super().__init__(self._interface_name, obj_path=obj_path)
+        super().__init__(interface_name=self._interface_name, obj_path=obj_path)
```

Comment:
There's no real point in this as `blueman.bluez.Base.BaseMeta.__call__` will receive the arguments of the call to `blueman.bluez.Adapter.Adapter.__init__` not to `blueman.bluez.Base.Base.__init__`.

If we want to enforce keyword arguments for the base class, `blueman.bluez.Base.Base.__init__` could be `__init__(self, *, interface_name: str, obj_path: str)` ([PEP 3102](https://www.python.org/dev/peps/pep-3102/))

Reply:
I'll check out PEP3102 and see if this does what I want. If so I'll make the changes you suggest.